json-guides/microprofile-config.json

```diff
@@ -101,28 +112,38 @@
                         "import javax.inject.Inject;",
                         "import org.eclipse.microprofile.config.Config;",
                         "import org.eclipse.microprofile.config.inject.ConfigProperty;",
-                        "", 
+                        "",    
                         "public class Music-download {",
+                        "",
                         "    @Inject",
                         "    private Config config;",
                         "",
                         "    @Inject @ConfigProperty(name=\"download-url\")",
-                        "    private String downloadUrl;",
-                        "}"                
+                        "    private String download-url;",
+                        "",
+                        "    public void download() {",
+                        "        downloadService(download-url);",
+                        "    }",
+                        "}"               
                     ],
                     "readonly": [
                        {
                             "from": "1",
-                            "to": "9"
+                            "to": "10"
                        },
                        {
-                            "from": "11",
-                            "to": "12"
+                            "from": "12",
+                            "to": "17"
                        }
                     ],
-                    "save": true,
-                    "callback": "(function test(editor) {microprofileConfigCallBack.listenToEditorForInjectConfig(editor); })"
-                    
+                    "save": false,
+                    "callback": "(function test(editor) {microprofileConfigCallBack.listenToEditorForInjectDefaultConfig(editor); })"                   
+                },
+                {
+                    "displayType":"webBrowser",
+                    "url": "https://music.com/download",
+                    "browserContent": "/guides/iguide-microprofile-config/html/interactive-guides/microprofile-config/download-from-injection.html",
```

Comment:
The browserContent should be the error exception from the step before to begin with.